calebgmahala/SCLR
# [Resolves #15] Create Floor Entity

# [Resolves #15] Create Floor Entity
Create floor entity and add easy render function

## Changes
- Added Floor entity
- Added function to render entities from string to Entity
- Moved demo functions into controllers
- Update demo app
- Added controllers directory to typedoc
- Added `buildParent` function to game entities

## Test
- Run application in development mode
- Observe how items are rendered
